lib/passes/PluginInfo.cpp

```diff
@@ -0,0 +1,42 @@
+#include <kllvm/codegen/MustTailDeadArgElimination.h>
+#include <kllvm/codegen/RemoveDeadKFunctions.h>
+#include <kllvm/codegen/SetVisibilityHidden.h>
+
+using namespace kllvm;
+
+/* New PM Registration */
+llvm::PassPluginLibraryInfo get_kllvm_plugin_info() {
+  return {
+      LLVM_PLUGIN_API_VERSION, "SetVisibilityHidden", LLVM_VERSION_STRING,
```

Comment:
This likely needs to be changed to something more general than `"SetVisibilityHidden"`.

Reply:
Good catch, I will fix this first thing tomorrow.